content/techniques/mongo.md

```diff
@@ -253,6 +253,27 @@ To inject a given `Connection` to a custom provider (for example, factory provid
 }
 ```
 
+While if you use the `@InjectModel` decorator you have to pass the connection name as an argument in addition to the name of your Mongodb collection like this:
+
+```typescript
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model } from 'mongoose';
+import { Cat, CatDocument } from './schemas/cat.schema';
+
+@Injectable()
+export class CatsService {
+  constructor(
+    @InjectModel(Cat.name, 'name_connection')
+    private catsModel: Model<CatDocument>
```

Comment:
I think this snippet:

```
@InjectModel(Cat.name, 'name_connection')
private catsModel: Model<CatDocument>
```
would be sufficient.

Everything else is irrelevant

Reply:
I opted for `way 1` i think it is more useful for beginners or for those approaching NestJS for the first time